STM32/LightController/Core/Src/LightController.c

```diff
@@ -259,4 +365,51 @@ void LightControllerInit(TIM_HandleTypeDef *htim2, TIM_HandleTypeDef *htim5, WWD
 void LightControllerLoop(const char* bootMsg)
 {
     CAhandleUserInputs(&caProto, bootMsg);
+
+    if (teststate == true){
+    
+    // Turn on every colour for 2 second during the test
+        switch(partystate){  
```

Comment:
Typically we add a "default" case. This catches if the state variable is wrong somehow. in this situation, I would go to teststate = false;\partystate = OFF; by default.

Reply:
Done